boto3/ec2/deletetags.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from boto3.utils import inject_attribute
+from boto3.resources.model import Action
+
+
+def inject_delete_tags(event_name, class_attributes, **kwargs):
```

Comment:
Not sure if it is a good idea yet, but how about you generate a lazy-loaded docstring and attach it to the method something like this: https://github.com/boto/boto3/blob/develop/boto3/resources/factory.py#L464 in the ``inject_delete_tags``.

I like the idea of modeling it as an actual action so that the documenter will pick it up and document it as a normal method, but I am not a big fan of having to add a new event and register two different handlers to complete the feature. It would be cool if the lazy-loaded docstrings can get flushed out in ``document_custom_method()`` so that we do not need to add a new event, we get docstrings for free, and we do not have to add two handlers everytime a resource model is incorrect and we have to add a customization to bridge some logic. Let me know what you think or if it is doable?